mava/networks.py

```diff
@@ -159,6 +163,52 @@ def __call__(self, obs_embedding: chex.Array, observation: Observation) -> tfd.I
         )
 
 
+class RecQNetwork(nn.Module):
+    """Recurrent Q-Network."""
+
+    pre_torso: nn.Module
+    post_torso: nn.Module
+    num_actions: int
```

Comment:
Adding in centralised critic functionality
```suggestion
    num_actions: int
    centralised_critic: bool = False
```